<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TableStateDemo</class>
 <widget class="QTabWidget" name="TableStateDemo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1053</width>
    <height>684</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TableStateDemo</string>
  </property>
  <property name="currentIndex">
   <number>3</number>
  </property>
  <widget class="QWidget" name="page_tableview">
   <attribute name="title">
    <string/>
   </attribute>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <property name="spacing">
     <number>5</number>
    </property>
    <item row="0" column="0">
     <widget class="QPushButton" name="bttn_back">
      <property name="text">
       <string>&amp;back</string>
      </property>
      <property name="shortcut">
       <string>Backspace</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="index_name">
      <property name="text">
       <string>Index</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QTableView" name="_view">
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="dragDropMode">
       <enum>QAbstractItemView::DragDrop</enum>
      </property>
      <property name="defaultDropAction">
       <enum>Qt::MoveAction</enum>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectItems</enum>
      </property>
      <property name="iconSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="tab">
   <attribute name="title">
    <string>Page</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="3" column="0" colspan="4">
        <widget class="QTableView" name="view_players">
         <property name="styleSheet">
          <string notr="true">QAbstractItemView {
    background-color: transparent;
}</string>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::DoubleClicked</set>
         </property>
         <property name="dragEnabled">
          <bool>true</bool>
         </property>
         <property name="dragDropMode">
          <enum>QAbstractItemView::DragDrop</enum>
         </property>
         <property name="defaultDropAction">
          <enum>Qt::MoveAction</enum>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>60</number>
         </attribute>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QFrame" name="frame_2">
         <property name="styleSheet">
          <string notr="true">QLabel {
    border: 1px solid;
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="nickname">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>10</y>
            <width>151</width>
            <height>20</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>153</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="stack">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>40</y>
            <width>151</width>
            <height>20</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>153</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="avatar">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>10</y>
            <width>50</width>
            <height>50</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="SvgWidget" name="svgwidget">
         <property name="sourceFile">
          <url>
           <string>qrc:/poker_client/data/textures/svg/cards_03/A_diamonds.svg</string>
          </url>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QFrame" name="fr">
         <property name="styleSheet">
          <string notr="true">QLabel {
    border: 1px solid #191919;
    border-radius: 5px;
    background-color: #fafafa;
	font: 22px &quot;Segoe UI&quot;;
}</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>5</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="table_card_1">
            <property name="minimumSize">
             <size>
              <width>53</width>
              <height>71</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>53</width>
              <height>71</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">color: #191919;</string>
            </property>
            <property name="text">
             <string>♠J</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="table_card_2">
            <property name="minimumSize">
             <size>
              <width>53</width>
              <height>71</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>53</width>
              <height>71</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">color: #191919;</string>
            </property>
            <property name="text">
             <string>♠2</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="table_card_3">
            <property name="minimumSize">
             <size>
              <width>53</width>
              <height>71</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>53</width>
              <height>71</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">color: #a80011;</string>
            </property>
            <property name="text">
             <string>♦Q</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="table_card_4">
            <property name="minimumSize">
             <size>
              <width>53</width>
              <height>71</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>53</width>
              <height>71</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">color: #a80011;</string>
            </property>
            <property name="text">
             <string>♥9</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="table_card_5">
            <property name="minimumSize">
             <size>
              <width>53</width>
              <height>71</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>53</width>
              <height>71</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel {
    
	background-color: qlineargradient(spread:reflect, x1:0, y1:1, x2:0.243574, y2:0.722, stop:0.655629 rgba(219, 219, 219, 234), stop:0.701987 rgba(0, 199, 82, 255), stop:0.837748 rgba(0, 121, 41, 255), stop:0.854305 rgba(0, 199, 82, 255));
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="bttn_reset">
         <property name="text">
          <string>reset</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" rowspan="3">
        <widget class="QTreeView" name="view_cards">
         <property name="styleSheet">
          <string notr="true">QAbstractItemView {
    background-color: transparent;
}</string>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::DoubleClicked</set>
         </property>
        </widget>
       </item>
       <item row="0" column="3" rowspan="3">
        <widget class="QListView" name="listView">
         <property name="styleSheet">
          <string notr="true">QListView::item {
    border: 1px solid blue;
width:48px;
height:70px;
}</string>
         </property>
         <property name="dragDropMode">
          <enum>QAbstractItemView::NoDragDrop</enum>
         </property>
         <property name="defaultDropAction">
          <enum>Qt::TargetMoveAction</enum>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="iconSize">
          <size>
           <width>92</width>
           <height>89</height>
          </size>
         </property>
         <property name="movement">
          <enum>QListView::Static</enum>
         </property>
         <property name="flow">
          <enum>QListView::LeftToRight</enum>
         </property>
         <property name="resizeMode">
          <enum>QListView::Fixed</enum>
         </property>
         <property name="spacing">
          <number>10</number>
         </property>
         <property name="gridSize">
          <size>
           <width>52</width>
           <height>74</height>
          </size>
         </property>
         <property name="viewMode">
          <enum>QListView::IconMode</enum>
         </property>
         <property name="uniformItemSizes">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="tab_2">
   <attribute name="title">
    <string>Страница</string>
   </attribute>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>431</width>
      <height>351</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    border: 1px solid;
}</string>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <property name="spacing">
      <number>5</number>
     </property>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="label_1">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="5">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3" colspan="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4" rowspan="3">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="3">
      <widget class="QLabel" name="label_0">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" rowspan="3">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2" rowspan="3">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3" rowspan="4">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" rowspan="3">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>GroupBox</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <property name="spacing">
         <number>5</number>
        </property>
        <item row="2" column="3" rowspan="3" colspan="3">
         <widget class="QLabel" name="label_20">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" rowspan="2">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" rowspan="4">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="5">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1" colspan="2">
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="3">
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4" colspan="2">
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2" colspan="2">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="tab_3">
   <attribute name="title">
    <string>Page</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout_6">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <property name="spacing">
     <number>5</number>
    </property>
    <item row="3" column="1">
     <widget class="QTreeView" name="tree_4"/>
    </item>
    <item row="1" column="1">
     <widget class="QTreeView" name="tree_2"/>
    </item>
    <item row="3" column="0">
     <widget class="QTreeView" name="tree_3"/>
    </item>
    <item row="1" column="0">
     <widget class="QTreeView" name="tree_1"/>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_19">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_21">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>EntityWidget</class>
   <extends>QWidget</extends>
   <header>plugins.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SvgWidget</class>
   <extends>EntityWidget</extends>
   <header>plugins.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local">../../basewidgets/basewidgets/svgwidget.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
